style.css

```diff
@@ -21,6 +21,48 @@ li {
   -----Header-----
 */
 
+.overlay-menu {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 30px;
+  width: 100%;
+  height: 100%;
+  color: white;
+  background-color: rgba(64, 83, 252, 0.7);
+  backdrop-filter: blur(7px);
+}
+
+.overlay-menu .content {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  margin: auto;
+  padding: 20px;
```

Comment:
To make the design match the Figma template, I recommend removing this padding property